inst/assets/BS4/pkgdown.css

```diff
@@ -278,6 +278,7 @@ nav[data-toggle="toc"] .nav-link.active:focus {
   color: inherit;
   border-left: none;
   background-color: #eee;
+  font-weight: inherit;
```

Comment:
Still looks bold on https://6054d322d05c89007befc56d--pkgdown-dev.netlify.app/articles/pkgdown.html

Reply:
Not... on my machine :thinking: 

![image](https://user-images.githubusercontent.com/8360597/111815365-de13ff80-88db-11eb-986c-86595bf14db4.png)